.nupic_modules

```diff
@@ -1,3 +1,3 @@
 # Default nupic.core dependencies (override in optional .nupic_config)
 NUPIC_CORE_REMOTE = 'git://github.com/numenta/nupic.core.git'
-NUPIC_CORE_COMMITISH = '706b5682b35e0f78ce1a36c0e7100d109b04ed54'
```

Comment:
this change should be dropped before merging (or core.SHA updated to this value )